docs/tutorials/ai/durable-ai-agent/index.md

```diff
@@ -0,0 +1,6078 @@
+
+# Tell hatchling what to include
+[tool.hatch.build.targets.wheel]
+packages = ["activities", "api", "models", "prompts", "shared", "tools", "workflows"]
+```
+
+This configures `uv` as to which packages to include and enable for execution.
+You will create these packages later in the tutorial.
+
+<details>
+
+<summary>
+The <code>pyproject.toml</code> is complete and will need no more revisions. You can review the complete file and copy the code here
+</summary>
+
+[pyproject.toml](https://github.com/temporal-community/tutorial-temporal-ai-agent/blob/main/pyproject.tom)
+
+```ini
+[project]
+name = "temporal-ai-agent"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.9"
+dependencies = [
+    "python-dotenv>=1.0.0",
+    "fastapi>=0.115.12",
+    "jinja2>=3.1.6",
+    "litellm>=1.72.2",
+    "stripe>=12.2.0",
+    "temporalio>=1.12.0",
+    "uvicorn>=0.34.3",
+    "requests>=2.32.4",
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+# Tell hatchling what to include
+[tool.hatch.build.targets.wheel]
+packages = ["activities", "api", "models", "prompts", "shared", "tools", "workflows"]
+
+```
+</details>
+
+Next, create a `.env` file to store your configuration:
+
+```command
+touch .env
+```
+
+Next, copy the following configuration to your `.env` file.
+
+```ini
+# LLM Configuration
+LLM_MODEL=openai/gpt-4o
+LLM_KEY=YOUR_OPEN_AI_KEY
+
+# Set if the user should click a Confirm button in the UI to allow the tool
+# to execute
+SHOW_CONFIRM=True
+
+# Temporal Configuration
+TEMPORAL_ADDRESS=localhost:7233
+TEMPORAL_NAMESPACE=default
+TEMPORAL_TASK_QUEUE=agent-task-queue
+
+# (Optional) - Uncomment both lines and set RAPIDAPI_KEY if you plan on 
+# using the real flights API
+# RAPIDAPI_KEY=YOUR_RAPID_API_KEY
+# RAPIDAPI_HOST_FLIGHTS=sky-scrapper.p.rapidapi.com
+
+# (Optional) - Uncomment and set STRIPE_API_KEY if you plan on using the Stripe
+# API to generate a fake invoice
+# STRIPE_API_KEY=YOUR_STRIPE_API_KEY
+
+# Uncomment if connecting to Temporal Cloud using mTLS (not needed for local dev server)
+# TEMPORAL_TLS_CERT='path/to/cert.pem'
+# TEMPORAL_TLS_KEY='path/to/key.pem'
+
+# Uncomment if connecting to Temporal Cloud using API key (not needed for local dev server)
+# TEMPORAL_API_KEY=abcdef1234567890
+```
+
+Once copied, replace `YOUR_OPEN_API_KEY` with your OpenAI API key.
+Setting `SHOW_CONFIRM=True` requires the user to confirm each tool prior to it being executed.
+This will allow you to see what the agent is doing step by step.
+These are the only two mandatory variables to set.
+This tutorial provides both an ability to create pseudo tools that perform simulations, or tools that use external APIs to achieve their goals.
+If you plan on using the [RapidAPI SkyScraper API](#optional) to look up flight data or the [Stripe API](#optional) to generate an invoice, you can uncomment these lines and provide the API keys here.
+
```

Comment:
This whitespace doesn't seem to be rendered which makes the preceding paragraph too long... maybe just throw a `<br/>` in here, idk what Docusaurus is doing.